src/iOS/Xamarin.iOS/Samples/Security/TokenSecuredChallenge/TokenSecuredChallenge.cs

```diff
@@ -300,20 +300,22 @@ public LoginOverlay(CGRect frame, nfloat transparency, UIColor color, string url
             nfloat buttonWidth = 60;
 
             // Get the total height and width of the control set (five rows of controls, four sets of space).
-            nfloat totalHeight = (5 * controlHeight) + (4 * rowSpace);
+            nfloat totalHeight = 5 * controlHeight + 4 * rowSpace;
```

Comment:
IMO this was easier to read with parens. What is gained by removing them? Why make the reader check operator precedence in their head?